themes/_themebase.scss

```diff
@@ -217,9 +217,14 @@ $hover-contrast-change: 2%;
   --delta-btn-tab-selected-bg: #{$colorPrimary};
   --delta-btn-tab-selected-text: #{$textPrimary};
 
+  /** MessageDetails Colors **/
+  --MessageDetails-ReadReceipt-Bg: #{changeContrast($bgPrimary, 10%)};
+  --MessageDetails-ReadReceipt-Date: #{changeContrast($textPrimary, 30%)};
```

Comment:
IDK about introducing these two variables on a theme-level. Sounds overly specific.

Reply:
let's better discuss this practice in https://github.com/deltachat/deltachat-desktop/issues/3018

I think overly specific vars are better than say reusing chatlist hover color somewhere in an unrelated component.